src/SonarScanner.MSBuild.Tasks/Targets/SonarQube.Integration.targets

```diff
@@ -239,7 +239,8 @@
     </PropertyGroup>
     <Message Condition="$(tmpSonarIsTestFileByNameResult)=='true'" Importance="normal" Text="Sonar: ($(MSBuildProjectFile)) project is evaluated as a test project based on the project name" />
 
-    <Message Importance="normal" Text="Sonar: ($(MSBuildProjectFile)) Project categorized. SonarQubeTestProject=$(SonarQubeTestProject)" />
+    <Message Importance="normal" Condition="$(SonarQubeTestProject)=='false'" Text="Sonar: ($(MSBuildProjectFile)) Project categorized. SonarQubeTestProject=$(SonarQubeTestProject)." />
+    <Message Importance="normal" Condition="$(SonarQubeTestProject)=='true'"  Text="Sonar: ($(MSBuildProjectFile)) Project categorized. SonarQubeTestProject=$(SonarQubeTestProject). Your SonarQube/SonarCloud project will not have results." />
```

Comment:
This renders one or the other. Alternative solution could be separate message on second line.

Reply:
I think the message is confusing.  It would be simpler to say `This MSBuild project will not be analyzed`.

Also, is it the case that third-party Roslyn analyzers will still be run against the project? (e.g. the XUnit/NUnit analzers?) If so, it would be more accurate to say something like `The Sonar analyzers will not executed against this MSBuild project`.

Reply:
Third-party analyzers are also removed. Only CS* compilation warnings will survive. This behavior is documented here (and we've verified the behavior): 
https://github.com/SonarSource/sonar-scanner-msbuild/wiki/Analysis-of-product-projects-vs.-test-projects#analysis-of-test-projects

Reply:
I also believe using `SonarQube/SonarCloud project` is confusing. The SQ project can be equivalent to the MSBuild solution. The MSBuild solution is just a module (from the scanner point of view) int the bigger SQ project.

+1 for `This MSBuild project will not be analyzed.`

Reply:
also, following up my other comment, I suggest to use `Project categorized as TEST project. This MSBuild project will not be analyzed.`